src/Stratis.Bitcoin.Features.ColdStaking/ColdStakingFeature.cs

```diff
@@ -111,11 +115,11 @@ public class ColdStakingFeature : BaseWalletFeature
             this.nodeSettings = nodeSettings;
             this.walletSettings = walletSettings;
 
-            nodeStats.RemoveStats(StatsType.Component, typeof(WalletFeature).Name);
-            nodeStats.RemoveStats(StatsType.Inline, typeof(WalletFeature).Name);
+            //nodeStats.RemoveStats(StatsType.Component, typeof(WalletFeature).Name);
```

Comment:
Why was this commented out?

Reply:
![image](https://user-images.githubusercontent.com/29645989/65768676-f1c22800-e174-11e9-9931-8a21188fb828.png)